components/blitz/src/omero/gateway/Gateway.java

```diff
@@ -1504,7 +1510,7 @@ public void shutDownDerivedConnector(SecurityContext ctx) throws Exception {
         try {
             c.closeDerived(isNetworkUp(true));
         } catch (Throwable e) {
-            new Exception("Cannot close the derived connectors", e);
+            throw new Exception("Cannot close the derived connectors", e);
```

Comment:
I'll remove the whole try catch block, `c.closeDerived(isNetworkUp(true))` actually can't throw an exception anyway.